nipype/interfaces/spm/model.py

```diff
@@ -45,52 +45,49 @@ class Level1DesignInputSpec(SPMCommandInputSpec):
         field="timing.RT", desc="Interscan interval in secs", mandatory=True
     )
     microtime_resolution = traits.Int(
-        field="timing.fmri_t", desc=("Number of time-bins per scan " "in secs (opt)")
+        field="timing.fmri_t", desc=("Number of time-bins per scan in secs (opt)")
     )
     microtime_onset = traits.Float(
         field="timing.fmri_t0",
-        desc=("The onset/time-bin in seconds for " "alignment (opt)"),
+        desc=("The onset/time-bin in seconds for alignment (opt)"),
     )
     session_info = traits.Any(
         field="sess",
-        desc=("Session specific information generated " "by ``modelgen.SpecifyModel``"),
+        desc=("Session specific information generated by ``modelgen.SpecifyModel``"),
         mandatory=True,
     )
     factor_info = traits.List(
         traits.Dict(traits.Enum("name", "levels")),
         field="fact",
-        desc=("Factor specific information " "file (opt)"),
+        desc=("Factor specific information file (opt)"),
     )
     bases = traits.Dict(
         traits.Enum("hrf", "fourier", "fourier_han", "gamma", "fir"),
         field="bases",
-        desc="""
-            dict {'name':{'basesparam1':val,...}}
-            name : string
-                Name of basis function (hrf, fourier, fourier_han,
-                gamma, fir)
-
-                hrf :
-                    derivs : 2-element list
-                        Model  HRF  Derivatives. No derivatives: [0,0],
-                        Time derivatives : [1,0], Time and Dispersion
-                        derivatives: [1,1]
-                fourier, fourier_han, gamma, fir:
-                    length : int
-                        Post-stimulus window length (in seconds)
-                    order : int
-                        Number of basis functions
+        desc="""\
+Dictionary names of the basis function to parameters:
+
+    * hrf
+
+        * derivs -- (2-element list) Model  HRF  Derivatives. No derivatives: [0,0],
+            Time derivatives : [1,0], Time and Dispersion derivatives: [1,1]
+
+    * fourier, fourier_han, gamma, or fir:
+
+        * length -- (int) Post-stimulus window length (in seconds)
+        * order -- (int) Number of basis functions
+
```

Comment:
The formatting of this looks a little weird: https://2093-17021985-gh.circle-artifacts.com/0/tmp/src/nipype/doc/_build/html/api/generated/nipype.interfaces.spm.model.html#level1design

Reply:
I think this is a bit better: https://2102-17021985-gh.circle-artifacts.com/0/tmp/src/nipype/doc/_build/html/api/generated/nipype.interfaces.spm.model.html#level1design